io/plugins/eu.esdihumboldt.hale.io.geopackage.test/src/eu/esdihumboldt/hale/io/geopackage/GeopackageInstanceWriterTest.groovy

```diff
@@ -186,12 +185,13 @@ class GeopackageInstanceWriterTest {
 
 		InstanceCollection instances = new InstanceBuilder(types: schema).createCollection {
 			abc {
-				a(new BigInteger('123'))
+				a((new BigInteger('123')).intValue())
```

Comment:
I'm afraid that does not solve the problem. This test is here to make sure that `BigInteger`s are supported.
Were you able to find out what the underlying cause is?

Reply:
I googled for the solution and this was to convert them to int.
I'm yet to find the underlying cause. 
May be we merge this PR and I'll create a new one with some solution?

Reply:
@stempler, I have pushed my changes for BigInteger test case. The underlying cause of the failing test was case `overflow` when converting `Long.MAX_VALUE +1` into `Long`. So I modified the input as passing in the `Long.MAX_VALUE`.